### DISCUSSION TITLE
Welcome to spacemacs Discussions!

### DISCUSSION THREAD

**Discussion by syl20bnr** [syl20bnr/spacemacs#14383] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by dalanicolai** [comment]
Hello! Great to have this discussion feature here now. So let me start using it.

Although I came to discuss/question some other tiny point, let me start a discussion about spacemacs Discussions here on github first.
 I read above that this area can be used for:
- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members
- Welcome others and are open-minded. Remember that this is a community we build together muscle.

In my experience Gitter is (currently) used more for Spacemacs support questions and less for discussions about development.
So I hope this place will be used more for discussing development (e.g. poll opinions about some PR/keybindings etc. before making the PR, without opening an issue), and less for user support questions. Gitter works very fine for that. However, I think otherwise this Discussions area can have multiple "rooms/threads".

So maybe we can start a discussion about how to use this discussion area. See also my next comment...

**Comment by dalanicolai** [comment]
Hello again! I was using the latex layer, and I noticed that for the LSP backend, the reftex menu is found under `, R`, while without LSP backend this menu is bound to `, r`. Now with the LSP layer enabled, `, r` is not used at all. So I was wondering if there is any reason to bind it to `, R`. Otherwise I generally strongly prefer to bind it to lower case `r` (by default). (@smile13241324... It looks like you have implemented it)

Also if someone has a useful opinion about if this comment should actually be opened as an issue, then let us know :pray:

**Reply by smile13241324** [reply]
The lower case `r` bindings are already in use by LSPs standard `refactor` menu so I have moved them to upper case instead. I think I had to move a bunch more not to overwrite bindings for standard LSP capabilities.

I think the primary issue here is that the Latex layer is not using the standard bindings from CONVENTIONS.org causing a lot of conflicts when the standard bindings from LSP were introduced. However I did not dare to change these given that they are around for such a long time now so I have added this dynamic moving of bindings based on whether LSP is used or not.

**Reply by dalanicolai** [reply]
Ah okay thanks. Well, do you expect the latex LSP to get some refactor menu? I guess the reftex was bound to `r` because latex did not require a refactor menu. So personally, I would say we could bind `reftex` menu to lowercase `r`, until maybe some day latex will get a refactor menu.

**Reply by lebensterben** [reply]
TexLab has rename provider:
https://github.com/latex-lsp/texlab/blob/16d96faa921bd01234d070ba2c28f5eb5542ca89/src/rename/mod.rs

And LaTeX is a programming language so surely it requires refactoring.
